src/_cffi_src/openssl/ssl.py

```diff
@@ -634,8 +635,34 @@
 static const long Cryptography_HAS_TLSEXT_STATUS_REQ_TYPE = 1;
 static const long Cryptography_HAS_RELEASE_BUFFERS = 1;
 static const long Cryptography_HAS_OP_NO_COMPRESSION = 1;
+
+#ifdef OPENSSL_NO_TLS1_METHOD
+static const long Cryptography_HAS_TLSv1 = 0;
```

Comment:
This like this need to be present in _conditional.py

Reply:
I added it. These names seem wrong though. What do you think about renaming to match Cryptography_HAS_SSL3_METHOD ?

Reply:
Sure, consistency sounds good.